http://sourceware.org/ml/gdb-cvs/2013-06/msg00015.html

### src/gdb/ChangeLog	2013/06/04 12:53:33	1.15682
### src/gdb/ChangeLog	2013/06/04 13:02:15	1.15683
## -1,5 +1,9 @@
 2013-06-04  Gary Benson  <gbenson@redhat.com>
 
+	* NEWS: Update.
+
+2013-06-04  Gary Benson  <gbenson@redhat.com>
+
 	* objfiles.h (inhibit_section_map_updates): New function
 	declaration.
 	(resume_section_map_updates): Likewise.
Index: gdb-7.6/gdb/NEWS
===================================================================
--- gdb-7.6.orig/gdb/NEWS	2013-06-10 14:27:02.071184153 +0200
+++ gdb-7.6/gdb/NEWS	2013-06-10 14:28:34.391145339 +0200
@@ -4,6 +4,16 @@
 * Newly installed $prefix/bin/gcore acts as a shell interface for the
   GDB command gcore.
 
+* New remote packets
+
+qXfer:libraries-svr4:read's annex
+  The previously unused annex of the qXfer:libraries-svr4:read packet
+  is now used to support passing an argument list.  The remote stub
+  reports support for this argument list to GDB's qSupported query.
+  The defined arguments are "start" and "prev", used to reduce work
+  necessary for library list updating, resulting in significant
+  speedup.
+
 *** Changes in GDB 7.6
 
 * Target record has been renamed to record-full.
